### PR TITLE
Add GitHub Actions badge

### PR DESCRIPTION
**Pull request by tsuyoshicho**
Auxiliary PR.

Add GitHub Actions badge in README.
(and small fix)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # gjo
 
+[![GitHub Actoins status](https://github.com/skanehira/gjo/workflows/Go/badge.svg)](https://github.com/skanehira/gjo/actions)
```
Comment (k-takata) [inline README.md]: Actoins -> Actions

Comment (tsuyoshicho) [reply README.md]: Thanks...

**Comment by tsuyoshicho** [comment]
remove small fix.

This spaces use new p in markdown

**Comment by skanehira** [comment]
Thank you :)

**Comment by tsuyoshicho** [comment]
Thanks too.

**Pull request merged**